client/js/util.js

```diff
@@ -9,7 +9,7 @@ const KEYS = {
 };
 
 const nodeList = (nl) => {
-  return [...nl];
+  return Array.prototype.slice.call(nl || []);
```

Comment:
Babel's internal `_toConsumableArray` method was breaking in IE10 even though we're using the es2015 preset. I couldn't figure out why so reverted to the old-skool/working way.